src/index.js

```diff
@@ -1 +1,7 @@
-// export your functions here
+export const every = (array, test) => {
+  for (const element of array) {
+    if (!test(element)) return false;
```

Comment:
The syntax for every as per mdn is `arr.every(callback(element[, index[, array]])[, thisArg])` - so I reckon you'll want to pass through the element, index and the array to the callback.